best-practice-authorisation.md

```diff
@@ -249,7 +248,7 @@ The access token MUST be a JSON Web Signature (JWS) as defined by [RFC 7515](RFC
 JWAs MUST NOT be used.
 
 The JWS MUST be signed with `RSASSA-PKCS1-v1_5 using SHA-512`, meaning the value of the `alg` field in the
-token's JOSE header MUST be set to `RS512` as defined in [RFC 7518](RFC-7518). An example JOSE header would
+token's JOSE header MUST be set to `RS512` as defined in [RFC 7518](RFC-7518). An example JOSN header would
```

Comment:
What is "JOSN"? JOSE is JSON Object Signing and Encryption, defined by https://tools.ietf.org/html/rfc7515#section-2

Reply:
Thank you I did not know that was a thing

Reply:
To be honest, neither did I until I reviewed this spec!

Reply:
maybe add a link to it in your PR #22

Reply:
Yes, that's a good plan, but I will wait to get some feedback on #22, #23, #24, #25 before going further. I suspect the authors are busy with build-up to NAB at the moment.